x/cron/types/params.go

```diff
@@ -0,0 +1,96 @@
+package types
+
+import (
+	"fmt"
+	// errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	// sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+var (
+	DefaultSecurityAddress = []string{"aib1sdggcl7eanaanvcsmvars7l0unsge65wzjm3dc"} //assuming BECH32_PREFIX=aib, will need a Default admin to whitelist contract for cron operations
+
+	DefaultContractGasLimit uint64 = 1000000000
```

Comment:
This one does not be used in any places, please remove it

Reply:
removed